cmake/thirdparty/SetupSeracThirdParty.cmake

```
@@ -336,6 +335,19 @@ if (NOT SERAC_THIRD_PARTY_LIBRARIES_FOUND)
         set(PETSC_FOUND FALSE)
     endif()
 
+    #------------------------------------------------------------------------------
+    # SLEPC
+    #------------------------------------------------------------------------------
+    if(SLEPC_DIR)
+        serac_assert_is_directory(DIR_VARIABLE SLEPC_DIR)
+        include(${CMAKE_CURRENT_LIST_DIR}/FindSLEPc.cmake)
+        message(STATUS "SLEPc support is ON")
```

```suggestion
        serac_assert_find_succeeded(PROJECT_NAME SLEPC
                                                       TARGET PkgConfig::SLEPC
                                                       DIR_VARIABLE SLEPC_DIR)
        message(STATUS "SLEPc support is ON")
```

I added this helper function right before this PR.

---

Please fix the indentation... its hard in Github to get right.